packages/from-query/src/index.ts

```diff
@@ -410,7 +428,15 @@ const doIt: FromQuerySignature = (schema, query, typeMap = {}, providedOptions =
     } else {
       throw new Error(`Unsupported Definition ${def.kind}`);
     }
-  });
+  }).concat(
+    enumDeclarations.size ? [
```

Comment:
realized that enums were possibly being included more than once, this prevents that from happening